common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -347,8 +347,11 @@ def transcript(self, request, dispatch):
                     mimetype,
                     add_attachment_header=False
                 )
-            except NotFoundError:
-                log.exception('[Translation Dispatch] %s', self.location)
+            except NotFoundError as exc:
+                edx_video_id = clean_video_id(self.edx_video_id)
+                log.warning('[Translation Dispatch] %s: %s',
+                    self.location,
+                    exc if is_bumper else f'Transcript not found for {edx_video_id}')
```

Comment:
The quality failure is: 
```
./common/lib/xmodule/xmodule/video_module/video_handlers.py:353:21: E128 continuation line under-indented for visual indent
./common/lib/xmodule/xmodule/video_module/video_handlers.py:354:21: E128 continuation line under-indented for visual indent
```
Use this format:
```suggestion
                log.warning(
                    '[Translation Dispatch] %s: %s',
                    self.location,
                    exc if is_bumper else f'Transcript not found for {edx_video_id}'
                )
```